puppet/files/etc/apache2/conf.d/mozilla-kuma-apache.conf

```diff
@@ -14,11 +14,7 @@ Listen 8080
     SetEnv HTTPS on
     RequestHeader set X-Forwarded-Proto "https"
 
-    # Covering all pre-bug-702988 past-known server aliases
     ServerName developer-local.allizom.org
-    ServerAlias developer-kumadev.mozilla.org
-    ServerAlias developer-mdndev.mozilla.org
-    ServerAlias developer-dev.mozilla.org
     ServerAlias mdn-local.mozillademos.org
```

Comment:
`mdn-local.mozillademos.org` is used for serving demos locally under a different domain than the main one `developer-local.allizom.org`, similar to `mdn.mozillademos.org` in production.